src/helpers/appUpdateText.ts

```diff
@@ -0,0 +1,27 @@
+import { getDeviceLanguage } from "helpers/localeUtils";
+import { t } from "i18next";
+
+/**
+ * Helper function to get app update text from payload or fallback to translations
+ * @param appUpdateText - The app update text object with enabled and payload
+ * @returns The localized update text
+ */
+export const getAppUpdateText = (appUpdateText: {
```

Comment:
would you mind adding a video showing the text being updated through the feature flags? Maybe switching between short and long texts - thanks!